mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOps.td

```diff
@@ -207,6 +207,16 @@ def SYCL_GroupType : SYCL_Type<"Group", "group"> {
   let assemblyFormat = "`<` `[` $dimension `]` `,` `(` $body `)` `>`";
 }
 
+def SYCL_GetOpType : SYCL_Type<"GetOp", "get_op"> {
+  let assemblyFormat = "";
+}
```

Comment:
I'd say we need a `dataType` parameter here holding the data type.

Reply:
`GetOp` holds no data type.

Reply:
I was thinking on a scenario in which we wanted to have `getValue` or `operator()` as operations in the dialect and add verifiers for those. If you think that'll never be the case, we can just merge this.

Reply:
ic, it can be useful, added.